crates/ruma-client-api/src/r0/profile/get_avatar_url.rs

```diff
@@ -30,7 +30,16 @@ ruma_api! {
             feature = "compat",
             serde(default, deserialize_with = "ruma_serde::empty_string_as_none")
         )]
-        pub avatar_url: Option<MxcUri>
+        pub avatar_url: Option<MxcUri>,
+
+        /// The [BlurHash](https://blurha.sh) for the avatar pointed to by `avatar_url`.
+        ///
+        /// This uses the unstable prefix in
+        /// [MSC2448](https://github.com/matrix-org/matrix-doc/pull/2448).
+        #[serde(skip_serializing_if = "Option::is_none")]
+        #[cfg_attr(docsrs, doc(cfg(feature = "unstable-pre-spec")))]
```

Comment:
The `doc(cfg)` attribute is there, but the regular `cfg` attribute (without `cfg_attr`) is missing, to actually feature-gate this field.

Reply:
I've added the regular `cfg` attributes to all the fields I've added.
They compile fine in CI and `cargo clippy --all-features`. However `cargo check --features full` (or `cargo check --features client-api`) fails. I'm not sure why that would be the case.

The error message is:
```
error[E0425]: cannot find value `blurhash` in this scope
  --> crates/ruma-client-api/src/r0/profile/set_avatar_url.rs:49:13
   |
49 |         pub blurhash: Option<&'a str>,
   |             ^^^^^^^^ not found in this scope

error[E0560]: struct `set_avatar_url::IncomingRequest` has no field named `blurhash`
  --> crates/ruma-client-api/src/r0/profile/set_avatar_url.rs:49:13
   |
49 |         pub blurhash: Option<&'a str>,
   |             ^^^^^^^^ `set_avatar_url::IncomingRequest` does not have this field
   |
   = note: available fields are: `user_id`, `avatar_url`
   ```

Reply:
Oh, I need to look into this. Thanks for noticing.

Reply:
Is this still a problem, it almost seems like it's a macro issue but that doesn't make any sense since it works for some features but not others.

Reply:
Have not checked yet.

Reply:
I figured it out #659